docs/data/scheduler/event-properties/event-properties.md

```diff
@@ -15,6 +15,38 @@ githubLabel: 'scope: scheduler'
 This package is not published yet.
 :::
 
+## Event creation
+
+You can use the `eventCreation` prop to customize how newly created events are defined:
+
+### Disable event creation
+
+Pass `eventCreation={false}` to disable the event creation:
+
+```tsx
+<EventCreation eventCreation={false} />
+```
+
+### Custom default duration
+
+Pass a custom value to `eventCreation.duration` to change the default duration of newly created event:
+
+```tsx
+<EventCreation eventCreation={{ duration: 60 }} />
+```
+
+{{"demo": "EventCreationDuration.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Create event on click
+
+Set `eventCreation.interaction` to `"click"` to open the creation form when clicking a cell instead of double-clicking:
```

Comment:
Do you think we should switch `props.modal` to false on the popover when the interaction is "click"?

With `modal={true}`:

https://github.com/user-attachments/assets/fd9e799b-7ee4-4824-b079-be5264c95383

With `modal={false}`:

https://github.com/user-attachments/assets/c3ea18e9-dba6-441e-ad1f-27df5fbda71a

Of course the best UI would be to have the popover move without closing at all (this should be possible with the detached trigger)

Reply:
We can keep this for later, once we have the detached trigger :+1: